### PR TITLE
Change type filter

### PR DESCRIPTION
**Pull request by SpOOnman**

### CODE REVIEW

File: designer/submodules/packages/components/src/scenarios/scenariosView.tsx [hunk 1]
```diff
@@ -68,13 +68,12 @@ function Actions({ addScenario, addFragment }: { addScenario?: () => void; addFr
...
+    // @ts-ignore
     const valueLinker: ValueLinker<ScenariosFiltersModel> = useCallback(
         (setNewValue) => (id, value) => {
             switch (id) {
-                case "HIDE_SCENARIOS":
-                    return value && setNewValue("HIDE_FRAGMENTS", false);
-                case "HIDE_FRAGMENTS":
-                    return value && setNewValue("HIDE_SCENARIOS", false);
+                default:
+                    break;
```
Comment (gadomsky) [inline designer/submodules/packages/components/src/scenarios/scenariosView.tsx:76]: hmm, what does it do now?

File: designer/submodules/packages/components/src/scenarios/filters/otherOptionsStack.tsx [hunk 2]
```diff
@@ -23,18 +23,16 @@ export function OtherOptionsStack(): JSX.Element {
...
+                label={t("table.filter.SCENARIOS", "Scenarios")}
             />
             <FilterListItem
-                invert
                 color="default"
-                checked={getFilter("HIDE_FRAGMENTS") === true}
-                onChange={(checked) => setFilter("HIDE_FRAGMENTS", checked)}
-                label={t("table.filter.SHOW_FRAGMENTS", "Show fragments")}
+                checked={getFilter("TYPE", true)?.includes(ScenariosFiltersModelType.FRAGMENTS)}
+                onChange={() => setFilter("TYPE", xor([ScenariosFiltersModelType.FRAGMENTS], getFilter("TYPE", true)))}
```
Comment (gadomsky) [inline designer/submodules/packages/components/src/scenarios/filters/otherOptionsStack.tsx:34]: can we extract 
```js
const getFilterType = () => getFilter("TYPE", true);
```
?

File: designer/submodules/packages/components/src/scenarios/scenariosView.tsx (hunk 1 above)
Comment (JulianWielga) [reply designer/submodules/packages/components/src/scenarios/scenariosView.tsx:76]: probably remains as placeholder

Comment (SpOOnman) [reply designer/submodules/packages/components/src/scenarios/scenariosView.tsx]: I've removed id.

File: designer/submodules/packages/components/src/scenarios/filters/otherOptionsStack.tsx (hunk 2 above)
Comment (SpOOnman) [reply designer/submodules/packages/components/src/scenarios/filters/otherOptionsStack.tsx:34]: Done

File: designer/submodules/packages/components/src/scenarios/filters/filtersPart.tsx [hunk 3]
```diff
@@ -36,15 +36,17 @@ export function FiltersPart({ withSort, isLoading, data = [] }: { data: RowType[
...
-
     const getLabel = useCallback(
         (name: keyof ScenariosFiltersModel, value?: string | number) => {
             switch (name) {
-                case "HIDE_FRAGMENTS":
-                    return t("table.filter.desc.HIDE_FRAGMENTS", "Fragments hidden");
-                case "HIDE_SCENARIOS":
-                    return t("table.filter.desc.HIDE_SCENARIOS", "Scenarios hidden");
+                case "TYPE":
+                    switch (value) {
```
[the remark this answers is not in the document]
Comment (SpOOnman) [reply designer/submodules/packages/components/src/scenarios/filters/filtersPart.tsx:43]: Ok :sunglasses:

**Pull request merged**